common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -52,6 +68,92 @@ def fetch_file_contents(repo, commit, path)
         response.body
       end
 
+      def commits(repo, branch_name = nil)
+        commits_path = "#{repo}/commits/" + branch_name.to_s
+        commits_path += "?pagelen=100"
```

Comment:
```suggestion
        commits_path = "#{repo}/commits/#{branch_name}?pagelen=100"
```

Idea: simplify with string interpolation.